inference-engine/src/low_precision_transformations/src/reshape.cpp

```diff
@@ -147,11 +147,15 @@ void reshapeDequantizationConstant(const std::shared_ptr<opset1::Reshape>& resha
 
 bool ReshapeTransformation::transform(TransformationContext& context, ngraph::pattern::Matcher &m) const {
     std::shared_ptr<opset1::Reshape> reshape = as_type_ptr<opset1::Reshape>(m.get_match_root());
-    if ((reshape == nullptr) || (!canBeTransformed(context, reshape))) {
+    if (NetworkHelper::onWeights(reshape)) {
```

Comment:
Please use NetworkHelper::isConstantPath instead of NetworkHelper::onWeights

Reply:
done, thanks